test/functional/page_objects/dashboard_page.js

```diff
@@ -48,6 +48,7 @@ export function DashboardPageProvider({ getService, getPageObjects }) {
       await retry.try(async () => {
         await this.showPanelEditControlsDropdownMenu();
         await testSubjects.click('dashboardPanelEditLink');
+        await PageObjects.header.waitUntilLoadingHasFinished();
```

Comment:
I wonder if this should be after the check that makes sure the visualization page was loaded, rather than before.  I have some suspicions on this function, that it could possibly find the hidden loading indicator on the dashboard page, pass, then check that the visualization page is current, but then the loading indicator would appear again when the visualize page is loading.  

Also might be worthwhile to run the ci 4 or 5 times to ensure this stabilizes the tests

Reply:
@Stacey-Gammon I agree in doing more testing I missed doing that for this particular PR,  I will run the tests locally in a loop to ensure I can get consistent passing results.  In any case, this change does not do any harm, when you click edit visualization link from dashboard, the loading page indicator does show and the method should wait for it.  I will check if any other changes are required.